packages/integrations/markdoc/package.json

```diff
@@ -21,7 +21,7 @@
   "exports": {
     ".": "./dist/index.js",
     "./components": "./components/index.ts",
-    "./default-config": "./dist/default-config.js",
+    "./runtime": "./dist/runtime.js",
```

Comment:
Isn't this a breaking change? Genuine question

Reply:
Probably safer to make this a breaking minor! Neither of these exports are documented, so the change is internal. Though users could have been package splunking